static/app/views/organizationGroupDetails/groupEvents.spec.jsx

```diff
@@ -242,6 +244,29 @@ describe('groupEvents', function () {
       );
     });
 
+    it('does not make attachments request, when feature not enabled', async () => {
+      org = initializeOrg({
+        organization: {
+          features: ['performance-issues-all-events-tab'],
+        },
+      });
+
+      render(
+        <GroupEvents
+          organization={org.organization}
+          api={new MockApiClient()}
+          params={{orgId: 'orgId', projectId: 'projectId', groupId: '1'}}
+          group={group}
+          location={{query: {environment: ['prod', 'staging']}}}
+        />,
+        {context: routerContext, organization}
+      );
+      await waitForElementToBeRemoved(document.querySelector('div.loading-indicator'));
```

Comment:
This is interesting! All other tests in our suite do this:

```js
await waitForElementToBeRemoved(() => screen.queryByTestId('loading-indicator'));
```

Your approach also works because apparently `waitForElementToBeRemoved` scans upwards and find the parent element, and then checks for your selected element's disappearance!